llvm/include/llvm/IR/Instructions.h

```diff
@@ -2537,7 +2537,7 @@ class PHINode : public Instruction {
   /// edges that this phi node will have (use 0 if you really have no idea).
   static PHINode *Create(Type *Ty, unsigned NumReservedValues,
                          const Twine &NameStr = "",
-                         InsertPosition InsertBefore = nullptr) {
+                         InsertPosition InsertBefore = (BasicBlock*)nullptr) {
```

Comment:
Why is this necessary despite the `nullptr_t` overload on InsertPosition?

Reply:
Apparently it isn't (now removed). Not sure what I was doing before, but I thought I was getting a bunch of deprecated warnings out of here, which I now can't replicate.